metals/src/main/scala/scala/meta/internal/metals/debug/SourcePathProvider.scala

```diff
@@ -12,6 +12,10 @@ final class SourcePathProvider(
     buildTargets: BuildTargets,
     targets: List[BuildTargetIdentifier]
 ) {
+  def isWorkspacePath(path: AbsolutePath): Boolean = {
```

Comment:
Should we use this instead?
https://github.com/scalameta/metals/blob/59ca771c9d1c1f3ca3d34df0be84215bc86e4a40/metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala#L257